input/pagecontent/reference.md

```diff
@@ -1395,7 +1395,7 @@ The [CodeableReference](https://hl7.org/fhir/2020Feb/references.html#codeableref
   ```
   Instance:   LatexSubstanceExample
   InstanceOf: LatexSubstance
-  * code.concept = SCT#1003754000 "Natural rubber latex (substance)"
+  * code.concept = $SCT#1003754000 "Natural rubber latex (substance)"
```

Comment:
Can't comment on the line, but on 1436, this:
```
  * address.state from USPSTwoLetterAlphabeticCodes (extensible)
```
should be
```
  * address.state from $USPSTwoLetterAlphabeticCodes (extensible)
```

Reply:
USPSTwoLetterAlphabeticCodes  is also not a alias (see http://hl7.org/fhir/us/core/STU3.1/ValueSet-us-core-usps-state.json.html)

Reply:
Ah good point. I think in that case though we still may want to update to `UspsTwoLetterAlphabeticCodes`, since that is the true name: 
![image](https://user-images.githubusercontent.com/52747882/125321724-fd132300-e30a-11eb-95b3-fe593d5c0311.png)

Reply:
Only difference being the lack of caps on `USPS`.